source/common/api/win32/os_sys_calls_impl.cc

```diff
@@ -160,6 +160,19 @@ SysCallSizeResult OsSysCallsImpl::recvmsg(os_fd_t sockfd, msghdr* msg, int flags
   return {bytes_received, 0};
 }
 
+SysCallIntResult OsSysCallsImpl::recvmmsg(os_fd_t sockfd, struct mmsghdr* msgvec, unsigned int vlen,
+                                          int flags, struct timespec* timeout) {
+  NOT_IMPLEMENTED_GCOVR_EXCL_LINE;
+}
+
+bool OsSysCallsImpl::supportMmsg() const {
+#if ENVOY_MMSG_MORE
```

Comment:
Does this ever return true on windows? I kind of doubt it. I would probably just return false here. cc @wrowe

Reply:
ENVOY_MMSG_MORE is false on Windows, so it will never return true. But I would like ENVOY_MMSG_MORE to be the only switch to turn on/off recvmmsg rather than having different additional logic in different places